mock-up/scss/tech.scss

```
@@ -17,6 +17,7 @@
   padding: 10px;
   position: relative;
   width: 225px;
+  transition: after 0.5s ease;
```

```suggestion
  transition: right 0.5s ease, bottom 0.5s ease;
```
They are only for css properties. (not all)